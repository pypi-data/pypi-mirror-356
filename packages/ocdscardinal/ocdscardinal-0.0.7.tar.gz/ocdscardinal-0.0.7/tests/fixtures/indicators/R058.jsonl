{"ocid":"F","bids":{"details":[{"status":"valid","tenderers":[{"id":"W"}],"value":{"amount":1,"currency":"USD"}},{"status":"valid","tenderers":[{"id":"L"}],"value":{"amount":101,"currency":"USD"}}]},"awards":[{"status":"active","suppliers":[{"id":"W"}]}]}
{"ocid":"1","bids":{"details":[{"status":"valid","tenderers":[{"id":"W"}],"value":{"amount":1,"currency":"USD"}},{"status":"valid","tenderers":[{"id":"L"}],"value":{"amount":1,"currency":"USD"}}]},"awards":[{"status":"active","suppliers":[{"id":"W"}]}]}
{"ocid":"2","bids":{"details":[{"status":"valid","tenderers":[{"id":"W"}],"value":{"amount":1,"currency":"USD"}},{"status":"valid","tenderers":[{"id":"L"}],"value":{"amount":1,"currency":"USD"}}]},"awards":[{"status":"active","suppliers":[{"id":"W"}]}]}
{"ocid":"3","bids":{"details":[{"status":"valid","tenderers":[{"id":"W"}],"value":{"amount":1,"currency":"USD"}},{"status":"valid","tenderers":[{"id":"L"}],"value":{"amount":1,"currency":"USD"}}]},"awards":[{"status":"active","suppliers":[{"id":"W"}]}]}
{"ocid":"accept","bids":{"details":[{"status":"valid","tenderers":[{"id":"W"}],"value":{"amount":1,"currency":"USD"}},{"status":"valid","tenderers":[{"id":"L"}],"value":{"amount":1,"currency":"USD"}}]},"awards":[{"status":"active","suppliers":[{"id":"W"}]},{"status":"cancelled","suppliers":[{"id":"C"}]},{"status":"unsuccessful","suppliers":[{"id":"U"}]}]}
{"ocid":"pending award","bids":{"details":[{"status":"valid","tenderers":[{"id":"W"}],"value":{"amount":1,"currency":"USD"}},{"status":"valid","tenderers":[{"id":"L"}],"value":{"amount":101,"currency":"USD"}}]},"awards":[{"status":"active","suppliers":[{"id":"W"}]},{"status":"pending"}]}
{"ocid":"invalid award","bids":{"details":[{"status":"valid","tenderers":[{"id":"W"}],"value":{"amount":1,"currency":"USD"}},{"status":"valid","tenderers":[{"id":"L"}],"value":{"amount":101,"currency":"USD"}}]},"awards":[{"status":"active","suppliers":[{"id":"W"}]},{"status":"invalid"}]}
{"ocid":"invited bid","bids":{"details":[{"status":"invited","tenderers":[{"id":"W"}],"value":{"amount":1,"currency":"USD"}},{"status":"valid","tenderers":[{"id":"L"}],"value":{"amount":101,"currency":"USD"}}]},"awards":[{"status":"active","suppliers":[{"id":"W"}]}]}
{"ocid":"pending bid","bids":{"details":[{"status":"pending","tenderers":[{"id":"W"}],"value":{"amount":1,"currency":"USD"}},{"status":"valid","tenderers":[{"id":"L"}],"value":{"amount":101,"currency":"USD"}}]},"awards":[{"status":"active","suppliers":[{"id":"W"}]}]}
{"ocid":"withdrawn bid","bids":{"details":[{"status":"withdrawn","tenderers":[{"id":"W"}],"value":{"amount":1,"currency":"USD"}},{"status":"valid","tenderers":[{"id":"L"}],"value":{"amount":101,"currency":"USD"}}]},"awards":[{"status":"active","suppliers":[{"id":"W"}]}]}
{"ocid":"disqualified bid","bids":{"details":[{"status":"disqualified","tenderers":[{"id":"W"}],"value":{"amount":1,"currency":"USD"}},{"status":"valid","tenderers":[{"id":"L"}],"value":{"amount":101,"currency":"USD"}}]},"awards":[{"status":"active","suppliers":[{"id":"W"}]}]}
{"ocid":"non-price criteria","bids":{"details":[{"status":"valid","tenderers":[{"id":"W"}],"value":{"amount":2,"currency":"USD"}},{"status":"valid","tenderers":[{"id":"L"}],"value":{"amount":1,"currency":"USD"}}]},"awards":[{"status":"active","suppliers":[{"id":"W"}]}]}
{"ocid":"multiple awards","bids":{"details":[{"status":"valid","tenderers":[{"id":"W"}],"value":{"amount":1,"currency":"USD"}},{"status":"valid","tenderers":[{"id":"L"}],"value":{"amount":101,"currency":"USD"}}]},"awards":[{"status":"active","suppliers":[{"id":"W"}]},{"status":"active","suppliers":[{"id":"W"}]}]}
{"ocid":"multiple tenderers","bids":{"details":[{"status":"valid","tenderers":[{"id":"W"}],"value":{"amount":1,"currency":"USD"}},{"status":"valid","tenderers":[{"id":"L"},{"id":"M"}],"value":{"amount":101,"currency":"USD"}}]},"awards":[{"status":"active","suppliers":[{"id":"W"}]}]}
{"ocid":"multiple suppliers","bids":{"details":[{"status":"valid","tenderers":[{"id":"W"}],"value":{"amount":1,"currency":"USD"}},{"status":"valid","tenderers":[{"id":"L"}],"value":{"amount":101,"currency":"USD"}}]},"awards":[{"status":"active","suppliers":[{"id":"W"},{"id":"X"}]}]}
{"ocid":"cancelled tender","bids":{"details":[{"status":"valid","tenderers":[{"id":"W"}],"value":{"amount":1,"currency":"USD"}},{"status":"valid","tenderers":[{"id":"L"}],"value":{"amount":101,"currency":"USD"}}]},"awards":[{"status":"active","suppliers":[{"id":"W"}]}],"tender":{"status":"cancelled"}}
{"ocid":"no price comparison","bids":{"details":[{"status":"valid","tenderers":[{"id":"W"}],"value":{"amount":1,"currency":"USD"}},{"status":"valid","tenderers":[{"id":"L"}],"value":{"amount":101,"currency":"USD"}}]},"awards":[{"status":"active","suppliers":[{"id":"W"}]}],"tender":{"procurementMethodDetails":"NPC"}}
{"ocid":"exclude procurement method details","bids":{"details":[{"status":"valid","tenderers":[{"id":"W"}],"value":{"amount":1,"currency":"USD"}},{"status":"valid","tenderers":[{"id":"L"}],"value":{"amount":101,"currency":"USD"}}]},"awards":[{"status":"active","suppliers":[{"id":"W"}]}],"tender":{"procurementMethodDetails":"EXC"}}
