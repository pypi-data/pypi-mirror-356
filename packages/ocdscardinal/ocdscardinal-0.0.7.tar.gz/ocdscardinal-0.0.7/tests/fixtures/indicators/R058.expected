{"OCID":{"F":{"R058":100.0}},"Tenderer":{"W":{"R058":0.0}}}
